x/mint/types/minter_test.go

```diff
@@ -1,17 +1,53 @@
-package types
+package types_test
 
 import (
+	"errors"
 	"math/rand"
 	"testing"
 
 	sdkmath "cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/stretchr/testify/require"
+
+	"github.com/ignite/modules/x/mint/types"
 )
 
+func TestValidateMinter(t *testing.T) {
+	invalid := types.DefaultInitialMinter()
+	invalid.Inflation = sdk.NewDec(-1)
+
+	tests := []struct {
+		name   string
+		minter types.Minter
+		err    error
+	}{
+		{
+			name:   "should prevent validate for minter with negative inflation",
+			minter: invalid,
+			err:    errors.New("mint parameter Inflation should be positive, is -1.000000000000000000"),
```

Comment:
I would either replace by defining an error type to use `ErrorIs` or replace `err` with a boolean `isValid`, defining the error in the test case is not visual and hard to maintain